services/account.service.js

```diff
@@ -115,24 +93,14 @@ async function addOneAccount(accountDetails) {
  * @return {boolean} success or failure of changing account information
```

Comment:
This returns a promise which resolves to something else, please reflect that here.